asf_search/search/baseline_search.py

```diff
@@ -37,7 +36,8 @@ def stack_from_product(
 
     stack_params = get_stack_params(reference)
     stack = search(**stack_params, host=host, cmr_token=cmr_token, cmr_provider=cmr_provider)
-    calc_temporal_baselines(reference, stack)
+    stack, warnings = get_baseline_from_stack(reference=reference, stack=stack)
```

Comment:
It's probably nothing but the "get_baseline_from_stack" name is bugging me. It's more that it's altering the stack to include baseline? I don't know what a more accurate name would be though, I'm possibly just being overly pedantic.